package.json

```diff
@@ -40,23 +39,31 @@
 		"test": "node scripts/test.js --env=jsdom",
 		"build:css": "cpx \"./src/components/**/*.css\" ./lib",
 		"prebuild:lib": "rimraf lib",
-		"build:lib": "npm-run-all --parallel build:commonjs build:css gen:types gen:index build:copy-files",
+		"build:lib":
+			"npm-run-all --parallel build:commonjs build:css gen:types gen:index build:copy-files",
 		"build:copy-package-files": "node scripts/copyBuildFiles.js",
 		"build:typings": "cpx \"./src/components/**/*.d.ts\" ./lib",
 		"build:index": "cpx \"./src/components/index.js\" ./lib",
-		"build:copy-files": "npm-run-all --parallel build:copy-package-files build:typings build:index",
+		"build:copy-files":
+			"npm-run-all --parallel build:copy-package-files build:typings build:index",
 		"predeploy:docs": "npm run build:docs",
 		"deploy:docs": "gh-pages -d build",
-		"build:commonjs": "cross-env NODE_ENV=production babel ./src/components --out-dir ./lib --ignore spec.js",
+		"build:commonjs":
+			"cross-env NODE_ENV=production babel ./src/components --out-dir ./lib --ignore spec.js",
 		"prepublish": "npm run build:lib"
 	},
+	"prettier": {
+		"useTabs": true,
+		"tabWidth": 4,
+		"semi": true,
+		"singleQuote": true,
+		"bracketSpacing": true,
+		"jsxBracketSameLine": true,
+		"arrowParens": "always"
```

Comment:
This is not needed. If only one param is present in arrow funcs, don't include parens.